src/components/views/dialogs/ShareDialog.tsx

```diff
@@ -211,7 +211,6 @@ export default class ShareDialog extends React.PureComponent<IProps, IState> {
                         { matrixToUrl }
                     </a>
                     <a href={matrixToUrl} className="mx_ShareDialog_matrixto_copy" onClick={this.onCopyClick}>
-                        { _t('COPY') }
```

Comment:
Accessibility?

Reply:
Consistency. It's never been on the pre blocks

Reply:
we should be consistent towards accessibility, not away from it.

Reply:
Why would we do it here if it's not done for any icons elsewhere? Should I add a tooltip instead?

Reply:
We should do it in the other places instead of removing the functionality. Tooltips are probably not suitable - the icon needs to be legible to screen readers.

Reply:
Should I replace it with an accessible button with correct label instead then?

Reply:
Probably an AccessibleButtonWithTooltip, yea

Reply:
I could replace this one with an accessible button. Unfortunately the genius / mad hackery of the copy button in the code block made it too difficult to use an `AccessibleTooltipButton` there.

Reply:
Looks like this has been resolved.